### PR TITLE
feat(MIG-166): throw manifest not found error

### PR DESCRIPTION
**Pull request by abulseed**
Follow up PR to resolve an issue with the error code that is thrown if there is no manifest file provided to the import command.

### CODE REVIEW

File: packages/mdctl-axon-tools/lib/StudyDataTranslations.js [hunk 1]
```diff
@@ -34,6 +34,10 @@ class StudyDataTranslations {
       }
     }
 
+    if (!manifestData) {
+      throw Fault.create('mdctl.kManifestNotFound.error', { reason: 'There is no manifest set as parameter neither found in directory' })
```
Comment (imaimai86) [inline packages/mdctl-axon-tools/lib/StudyDataTranslations.js:38]: The error message seems ambiguous to me, what about this?
`No manifest found in the parameters and in the import directory`

Comment (abulseed) [reply packages/mdctl-axon-tools/lib/StudyDataTranslations.js:38]: Sounds the same to me

Comment (joejean) [reply packages/mdctl-axon-tools/lib/StudyDataTranslations.js:38]: I guess the grammatically correct version should be  "There is no manifest set as a parameter, nor is one found in the directory."

Comment (abulseed) [reply packages/mdctl-axon-tools/lib/StudyDataTranslations.js]: @imaimai86 @joejean just updated the error message as u guys suggested.

**Comment by alahawash** [comment]
Hi @abulseed we've released 1.0.71, can you branch out of it?

**Comment by abulseed** [comment]
@alahawash it seems that the changes related to MIG-166 doesn't exist on 1.0.71. Did we create this branch from `1.0.70`?

**Comment by alahawash** [comment]
> @alahawash it seems that the changes related to `MIG-166` doesn't exist on `1.0.71`. Did we create this branch from `1.0.70`?

Ohh that's awful, we've got to reconcile those branches. Can this MR wait?

**Comment by abulseed** [comment]
> > @alahawash it seems that the changes related to `MIG-166` doesn't exist on `1.0.71`. Did we create this branch from `1.0.70`?
> 
> Ohh that's awful, we've got to reconcile those branches. Can this MR wait?

yes no worries this can wait. Sounds good please let me know.

**Comment by alahawash** [comment]
Hi @abulseed , please rebase from `main` which should include all changes from 1.0.70 and 1.0.71. Going forward we'll be using `main` as our trunk

**Comment by abulseed** [comment]
@alahawash just rebased from `main` would u please review it now? i tried to minimize the changes appearing on this PR but failed to get the changes needed for `MIG-166`. 

are we sure we have merged `1.0.70` into main? i have searched on `main` for this file `packages/mdctl-axon-tools/lib/StudyDataTranslations.js` but couldn't find it.

**Comment by alahawash** [comment]
You are right @abulseed , when I merged before my local wasn't up-to-date. I did merge again and you should not have any issues now if you rebase from main. Sorry for the that

**Comment by abulseed** [comment]
@alahawash done! please review..

**Review by alahawash — COMMENTED** [review]
LGTM

**Review by alahawash — APPROVED** [review]

**Pull request merged**